### ISSUE TITLE
Could you summarize the arch differences from the original FCOS model?

### ISSUE THREAD

**Issue by floveqq** [Megvii-BaseDetection/DenseTeacher#19]
about the arch like loss,bn,dataaug and so on? thanks

**Comment by ZRandomize** [comment]
for supervised loss, cls loss and reg loss are same as in FCOS, using FocalLoss and IoU loss. The target of quality branch is replaced by IoU rather than centerness, using same BCE loss, following [this](https//arxiv.org/abs/2006.04388) work. We do not change bn. DataAug is not a part of model arch, we use augs same as in Unbiased Teacher.
